cranelift/codegen/src/isa/s390x/abi.rs

```diff
@@ -245,7 +248,7 @@ impl ABIMachineSpec for S390xMachineDeps {
             next_gpr += 1;
         }
 
-        for (i, mut param) in params.iter().copied().enumerate() {
+        for (i, mut param) in params.into_iter().copied().enumerate() {
```

Comment:
Do you still need the `.copied()` here?

Reply:
Ah, the `Item` is still a reference, never mind :)

Reply:
Yeah because otherwise it is references to params, and the s390x needs the actual value because it will re-assign the `param` local elsewhere in the loop.